core/src/main/resources/log4j4ibis.xml

```diff
@@ -112,6 +112,7 @@
 		<Logger name="org.springframework.security.ldap" level="WARN" />
 		<Logger name="org.frankframework.management.gateway" level="INFO" />
 		<Logger name="com.hazelcast" level="WARN" />
+		<Logger name="snakeyaml.introspector" level="ERROR" />
```

Comment:
SnakeYaml uses introspection to find bean getters/setters, which logs WARN messages. These are not harmful/meaningful.